libnavigation-ui/build.gradle

```diff
@@ -94,10 +92,6 @@ dependencies {
   testImplementation dependenciesList.robolectric
   testImplementation dependenciesList.json
 }
-
-apply from: 'javadoc.gradle'
```

Comment:
I believe this is needed for Javadoc

Reply:
We are now using dokka. Why do we need this?

Reply:
Yeah you're right I was thinking on JaCoCo 😅